cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/scorecompiler/ScoreCompilerImpl.java

```diff
@@ -105,7 +105,11 @@ public ExecutionPlan convert(Executable compiledExecutable) {
             Collection<Executable> executables = new ArrayList<>(filteredDependencies.values());
             executables.add(executable);
 
-            executionPlan.setSubflowsUUIDs(new HashSet<>(dependencies.keySet()));
+            HashSet<String> subflowsUuids = new HashSet<>(dependencies.keySet());
+            if (executable.getExternalExecutableDependencies() != null) {
```

Comment:
at this moment there is no way `getExternalExecutableDependencies()` is `null`, you could check if it's empty or at least add it as a extra condition for the if